cl/api/utils.py

```diff
@@ -736,7 +736,10 @@ def invert_user_logs(
 
     dates = make_date_str_list(start, end)
     for d in dates:
-        pipe.zrange(f"api:v3.user.d:{d}.counts", 0, -1, withscores=True)
+        for version in ["v3", "v4"]:
+            pipe.zrange(
+                f"api:{version}.user.d:{d}.counts", 0, -1, withscores=True
+            )
```

Comment:
I haven't read the code *super* carefully, but I'd be pretty surprised if this works. Before your change `results` variable would contain a list of results, one per day. And the `dates` variable would contain a list of dates, one per day. From there you can `zip` them together, and you're off to the races. 

But when you do the above, you wind up with a list of counts, with 2 values per day, if somebody has used both v3 and v4 on a given day. 

I suspect this is only working because you didn't test that? I think the solution is to add v3 and v4 together.

Reply:
Ah! My mock is super wrong then, appreciate the review. I'll read the docs on zrange and update.